### ISSUE TITLE
error: Uncaught (in promise) Http: connection closed before message completed 

### ISSUE THREAD

**Issue by LoganTann** [gr7d/aqua#100]
Sometimes, the following error appears randomly when I do a request : 

```
error: Uncaught (in promise) Http: connection closed before message completed
    at Object.respondWith (deno:ext/http/01_http.js:254:21)
```

Unfortunately, I don't have further information. I can't even reproduce sometimes.

The following github issue and linked threads might help : https://github.com/denoland/deno/issues/13203

**Comment by LoganTann** [comment]
It also happens a lot when returning a server sent event.

**Comment by LoganTann** [comment]
This issue is very blocking as it causes the server to completely crash @gr7d 

I believe we should handle the error at this line : https://github.com/gr7d/aqua/blob/main/shared.ts#L57C20-L57C20

**Comment by gr7d** [comment]
I'm currently trying to reproduce it. If you have any tips, I'd be thankful.

**Comment by predetermined** [comment]
Fix (#101) included in release v1.3.6. Thanks for your patience :)

**Comment by LoganTann** [comment]
Thank you too for maintaining this lib, it is very simple, and I'm using it in many projects !
